apps/comps/rpc/router/leaderboard/get-global.ts

```diff
@@ -0,0 +1,15 @@
+import { LeaderboardParamsSchema } from "@recallnet/services/types";
+
+import { base } from "@/rpc/context/base";
+
+/**
+ * Get global leaderboard across all relevant competitions
+ *
+ * Note: LeaderboardService.getGlobalLeaderboard never throws errors.
+ * It returns an empty response as fallback on any failure.
+ */
+export const getGlobal = base
+  .input(LeaderboardParamsSchema)
+  .handler(async ({ input, context }) => {
+    return await context.leaderboardService.getGlobalLeaderboard(input);
+  });
```

Comment:
No error handling like you've previously done?

Reply:
no, because `leaderboardService.getGlobalLeaderboard` explicitly swallows all errors and has an `emptyLeaderboardResponse` it returns in the case of any error

Reply:
anyway I put some basic boiler-plate error handling just for future proofing even though its not really needed atm